tests/unit/resources/nft_test.py

```diff
@@ -0,0 +1,54 @@
+"""Test Mint resource component."""
+from unittest import mock
+
+from magic_admin.resources.nft import NFT
+
+
+class TestNFT:  # noqa: WPS306
+    """Test Mint resource component."""
+
+    quantity = 2
+    destination_address = '0x3c15B0e0e00A9edD2Be824064f9C9C29fc136C4E'
+    token_id = 1
+    nft = NFT()
+    contract_id = 'bsdjfkn-sjknfskn-kjsnf'
+
+    def test_start_mint721(self):
+        """Test mint 721."""
+        self.nft.request = mock.Mock()
+        self.nft.start_mint721(
```

Comment:
I missed this in the PR review last time but found it when reviewing Ben's is it would be good to assert the response from these methods. If the response model ever does change, the tests would catch this

Reply:
Same goes for the other test below